ArcGISRuntimeSDKQt_CppSamples/Features/FilterByDefinitionExpressionOrDisplayFilter/FilterByDefinitionExpressionOrDisplayFilter.cpp

```diff
@@ -137,10 +137,10 @@ void FilterByDefinitionExpressionOrDisplayFilter::queryFeatureCountInCurrentExte
 void FilterByDefinitionExpressionOrDisplayFilter::resetDisplayFilterParams()
 {
   DisplayFilter* displayFilter = new DisplayFilter("Damaged Trees", "", this);
-  const QList<DisplayFilter*> available_filters{displayFilter};
+  const QList<DisplayFilter*> availableFilters{displayFilter};
 
-  ManualDisplayFilterDefinition* display_filter_defintion = new ManualDisplayFilterDefinition(displayFilter, available_filters, this);
-  m_featureLayer->setDisplayFilterDefinition(display_filter_defintion);
+  ManualDisplayFilterDefinition* displayFilterDefintion = new ManualDisplayFilterDefinition(displayFilter, availableFilters, this);
```

Comment:
Typo throughout. Please review.
```suggestion
  ManualDisplayFilterDefinition* displayFilterDefinition = new ManualDisplayFilterDefinition(displayFilter, availableFilters, this);
```